### PR TITLE
docs: fix quotes in Transaction#ntxid description

### PR DESCRIPTION
**Pull request by maxgiraldo**
Insert double quotes instead of a combination of single quote
and backtick.

### CODE REVIEW

**Comment by sgeisler** [comment]
Thanks for the PR! I think we'll have to wait for #541 to fix our CI before merging it, otherwise an owner had to override the check.

**Review by stevenroose — APPROVED** [review]
LGTM, thanks!

**Comment by sgeisler** [comment]
Can you rebase on master and force push please to make CI work?

**Review by apoelstra — APPROVED** [review]
ack 06dc0041c25c67432358119f663f7c944a59ba1d

**Pull request merged**
